OpenDreamRuntime/DreamConnection.cs

```diff
@@ -211,7 +211,7 @@ public void HandleMsgPromptResponse(MsgPromptResponse message)
 
         public void OutputDreamValue(DreamValue value) {
             if (value.Type == DreamValue.DreamValueType.DreamObject) {
-                DreamObject outputObject = value.GetValueAsDreamObject();
```

Comment:
Seems to me like replacing a (typecheck, Get) with a (typecheck, Must) instead of a single TryGet is defeating the whole point of this migration? Similar issue with the type checks in DMOpcodeHandlers